test/data/test_builtin_datasets.py

```diff
@@ -76,6 +76,29 @@ def test_raw_text_classification(self, info):
             self.assertEqual(torchtext.datasets.MD5[dataset_name], info['MD5'])
         del data_iter
 
+    @parameterized.expand(
+        load_params('raw_datasets.jsonl'),
+        name_func=_raw_text_custom_name_func)
+    def test_dataset_iterator_offsets(self, info):
+        dataset_name = info['dataset_name']
+        split = info['split']
+
+        # TODO: Change to this
```

Comment:
I'm going to change this, but left it like this for now to make sure the test is getting run